packages/frontend/src/view/pages/transaction/SpotForcedWithdrawalPage.tsx

```diff
@@ -53,7 +53,7 @@ function SpotForcedWithdrawalPage(props: SpotForcedWithdrawalPageProps) {
     <Page
       user={props.user}
       path={`/transactions/${props.transactionHash.toString()}`}
-      description="TODO: description"
+      description={`Details of the ${props.transactionHash.toString()} spot forced withdrawal transaction`}
```

Comment:
I suggest removing "spot". Withdrawal transactions or always spot. We use Spot/Perpetual prefixes in our code just to differentiate logic.